setup.py

```diff
@@ -21,12 +21,17 @@
     README = f.read()
 
 requires = [
-    "fastapi==0.65.2",
-    "uvicorn[standard]==0.13.4",
-    "PyYAML==5.4.1",
-    "typer==0.3.2",
-    "psycopg2==2.9.1",
-    "sqlalchemy==1.4.19",
+    'fastapi==0.65.2',
+    'uvicorn[standard]==0.13.4',
+    'PyYAML==5.4.1',
+    'typer==0.3.2',
+    'psycopg2==2.9.1',
+    'sqlalchemy==1.4.19',
+    'psycopg2==2.9.1',
+    'pyramid==2.0',
+    'pyramid_beaker==0.8',
+    'pyramid_openapi3==0.11',
+    'openapi-core<0.14',
```

Comment:
```suggestion
    'openapi-core==0.14',
```
Let's pin all versions.